src/TraceEvent/TraceLog.cs

```diff
@@ -154,11 +154,32 @@ public static TraceLog OpenOrConvert(string etlOrEtlxFilePath, TraceLogOptions o
         /// </summary>
         public static TraceLogEventSource CreateFromTraceEventSession(TraceEventSession session)
         {
+            return CreateFromTraceEventSession(session, 1000);
+        }
+
+        /// <summary>
+        /// From a TraceEventSession, create a real time TraceLog Event Source.   Like a ETWTraceEventSource a TraceLogEventSource
+        /// will deliver events in real time.   However an TraceLogEventSource has an underlying Tracelog (which you can access with
+        /// the .Log Property) which lets you get at aggregated information (Processes, threads, images loaded, and perhaps most
+        /// importantly TraceEvent.CallStack() will work.  Thus you can get real time stacks from events).
+        ///
+        /// Note that in order for native stacks to resolve symbolically, you need to have some Kernel events turned on (Image, and Process)
+        /// and only windows 8 has a session that allows both kernel and user mode events simultaneously.   Thus this is most useful
+        /// on Win 8 systems.
+        /// </summary>
+        /// <param name="realTimeDelayMSec">The delay in milliseconds between when an event is received in TraceLog and when it is dispatched to the real time event source.</param>
+        public static TraceLogEventSource CreateFromTraceEventSession(TraceEventSession session, int realTimeDelayMSec)
```

Comment:
Please rename to something like `minDispatchDelayMSec`.  From a quick read of the code, this is a minimum delay based on the time the event spent in the queue.  It may spend longer in the queue.

Reply:
Fixed